core/src/main/java/com/hcl/ecomm/core/servlets/ProductServlet.java

```diff
@@ -0,0 +1,65 @@
+package com.hcl.ecomm.core.servlets;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.hcl.ecomm.core.services.ProductService;
+import com.hcl.ecomm.core.utility.ProductUtility;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.*;
+
+@Component(service = Servlet.class, property = { "sling.servlet.paths=/bin/hclecomm/products",
+		"sling.servlet.method=" + HttpConstants.METHOD_GET, "sling.servlet.extensions=json" })
+public class ProductServlet extends SlingSafeMethodsServlet {
+
+	private static final long serialVersionUID = 4016057296495129474L;
+	private static final Logger LOG = LoggerFactory.getLogger(ProductServlet.class);
+
+	@Reference
+	ProductService productService;
+
+	protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
+			throws ServletException, IOException {
+
+		LOG.info("inside ProductServlet doGET method");
```

Comment:
@nitheshkumar-hcl : pl remove unnecessary logs

Reply:
Committed the changes as per the review comments